sklearn/datasets/_covtype.py

```diff
@@ -80,6 +82,8 @@ def fetch_covtype(*, data_home=None, download_if_missing=True,
         If True, returns ``(data.data, data.target)`` instead of a Bunch
         object.
 
+    as_frame : boolean, default=False.
+        If True, returns ``pandas.DataFrame`` instead of a Bunch object
```

Comment:
the result is a bunch object, I added .frame attribute to the bunch object. The frame attribute is a combined dataframe of X and y, with properly specified columns. Please see discussion #10733 for reference, this is the design from that thread of discussion.

Reply:
Yes, but here you say `If True, returns ``pandas.DataFrame`` instead of a Bunch object` which is not correct.

You can see the documentation and the implementation https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/datasets/_base.py#L297 as an example

Reply:
ok, got it. will adjust, thanks!